src/database/standalone-sqlite.ts

```diff
@@ -981,12 +981,19 @@ export class StandaloneSqliteDatabaseWorker {
     const hash = dataRow?.hash;
     const dataRoot = coreRow?.data_root;
 
+    const rootTransactionId =
+      coreRow?.root_transaction_id !== null &&
+      coreRow?.root_transaction_id !== undefined
```

Comment:
checking for `null` AND `undefined` because it can return null for transactions where `root_transaction_id` doesn't exist and return `undefined` for bundles/data items.